docs/source/array-api.rst

```diff
@@ -448,12 +448,10 @@ Other functions
 .. autofunction:: broadcast_arrays
 .. autofunction:: broadcast_to
 .. autofunction:: cbrt
-.. autofunction:: coarsen
```

Comment:
Thanks for working on this @mesejo! Overall this looks great, but would you mind removing the other `.. autofunction:: coarsen`, `.. autofunction:: concatenate`, etc. so that the API docstring order matches the order functions are listed in the `.. autosummary::` tables